articles/overview/quantum-computers-and-simulators.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Quantum computers and quantum simulators
+description: Learn about quantum hardware, quantum simulators, and how quantum operations work.
+author: bradben
+ms.author:  bradben
+ms.date: 04/22/2020
+ms.topic: overview
+uid: microsoft.quantum.overview.simulators
+---
+
+# Quantum computers and quantum simulators
+
+Quantum computers are still in the infancy of their development. The hardware and maintenance are expensive, and most systems are located in universities and research labs. The technology is advancing, though, and limited public access to some systems is available.
+
+Quantum simulators are software programs that run on classical computers and make it possible to run and test quantum programs in an environment that predicts how qubits will react to different operations.
+
+## Quantum hardware
+
+A quantum computer has three primary parts: the qubit storage, a method for transferring signals to the qubits, and a classical computer to run a program and send instructions.
+
+- The quantum particles used for qubits are fragile and highly sensitive to environmental interferences. For some methods of qubit storage, the unit that houses the qubits is kept at a temperature just above absolute zero to maximize their coherence.  
+- Signals are sent to the qubits using microwaves or lasers.
```

Comment:
It may be worth noting that microwave and laser control are great examples, but aren't exhaustive.